docs/delivery.md

```diff
@@ -0,0 +1,58 @@
+# Delivery
+
+## Fetching Delivery Data
+
+```python
+from crux import Crux
+
+conn = Crux()
+
+dataset = conn.get_dataset("A_DATASET_ID")
+delivery = dataset.get_delivery("A_DELIVERY_ID")
+
+if delivery.status == "DELIVERY_SUCCEEDED":
+    for file in delivery.get_data()
+        print(resource.id)
+        file.refresh() #Refresh to fetch the metadata
+        file.download("/tmp/{}".format(resource.name))
+```
+## Fetching Delivery Raw Data
+
+```python
+from crux import Crux
+
+conn = Crux()
+
+dataset = conn.get_dataset("A_DATASET_ID")
+delivery = dataset.get_delivery("A_DELIVERY_ID")
+
+for file in delivery.get_raw()
+    print(resource.id)
+    file.refresh() #Refresh to fetch the metadata
+    file.download("/tmp/{}".format(resource.name))
+```
+
+## Fetching Ingestions
+
+```python
+from crux import Crux
+
+conn = Crux()
+
+dataset = conn.get_dataset("A_DATASET_ID")
+ingestions = dataset.get_ingestions()
+
+ingestion = next(ingestions)
+
+# Get the DELTA data from the latest version of ingestion
+for file in ingestion.get_data(accepted_status=["DELIVERY_SUCCEEDED"])
+    print(resource.id)
+    file.refresh() #Refresh to fetch the metadata
+    file.download("/tmp/{}".format(resource.name))
+
+# Get the RAW data from the latest version of ingestion
+for file in ingestion.get_raw()
+    print(resource.id)
+    file.refresh() #Refresh to fetch the metadata
```

Comment:
Users shouldn't be forced to `refresh()`. I know I had a comment about how it would require getting metadata even if users don't need it. But looking into it, they need the metadata to do anything (e.g., `File.size` is needed to `File.download()`). So we might as well fetch the metadata as part of create the `File` instance.

Reply:
Thanks, updated the code.